castai/resource_node_configuration.go

```diff
@@ -286,6 +287,12 @@ func resourceNodeConfiguration() *schema.Resource {
 								},
 							},
 						},
+						"node_group_arn": {
```

Comment:
Add a const for the field name (like the others); makes it easier to list all fields at the top

Reply:
done